### ISSUE TITLE
HPOOL shows me without plot

### ISSUE THREAD

**Issue by cool2hatex** [hpool-dev/chia-miner#616]
I am online with proper timings but when I check website it shows i have no plot since 13:00. Also no revenue since that time but it also shows i am online with 22,68TB. I also set XProxy now also it shows I have no online or offline miner.

![image](https://user-images.githubusercontent.com/85751462/121704301-50137380-cadc-11eb-89c0-1e73426e8e57.png)
![image](https://user-images.githubusercontent.com/85751462/121704392-68838e00-cadc-11eb-84e0-a19408cffc99.png)

**Comment by chudleigh** [comment]
same

**Comment by cool2hatex** [comment]
So far my achievements are setting Xproxy and finding out username/password is admin/admin. I entered API key and set up a user. Nothing changed at all.

**Comment by mmaa-crypto** [comment]
same problem ,,,
you just entered your api key in xproxy??

**Comment by cool2hatex** [comment]
Yes. But now seems like even xproxy not working HPOOL came back to life.

![image](https://user-images.githubusercontent.com/85751462/121747473-25dba900-cb10-11eb-8462-416e0c21beb4.png)

**Comment by jedocampo** [comment]
mine is still the same

**Comment by teklab-de** [comment]
Same it drops over 30% but not coming back... this day i become 50-60% lower revenue? I have this problems since 8/9AM GTM+1

![screenshot-hpool com-2021 06 12-01_15_08](https://user-images.githubusercontent.com/9017746/121756814-b370c600-cb1b-11eb-9225-fbe566fe792b.png)

ok i go sleep now and wait one day. But i think i go back to solo minig. 0.33XCH/PB...

**Comment by archerallstars** [comment]
Mine go down also. I expected consistent revenue from pooling. Otherwise, I would rather go solo or not do it at all until the official pooling protocol is established.

EDIT: Now, my plots are showing online as they should. The problem was due to these 2 factors:
1. Currently, at least for me, I have to use the miner version 1.4.1. Otherwise, my plots won't go online. I messed around with 1.4.0 but it's no use. Please upgrade to 1.4.1.
2. If I want to add plots to the miner, I will have to stop the miner in my machine first and wait until it goes offline (around 30 mins), then I also delete this offline miner on the web. After that, I can add plots to the miner and then start the miner again. It seems their backend is broken currently. If you add plots while your miner is still online, your signed plots will go down in number and they will be stuck there no matter how many plots you added later on.
